cmd/convert.go

```diff
@@ -97,7 +145,7 @@ func init() {
 	convertCmd.Flags().MarkHidden("deployment")
 
 	// OpenShift only
-	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", true, "Generate an OpenShift deploymentconfig object")
+	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", false, "Generate an OpenShift deploymentconfig object")
```

Comment:
@surajssd validateControllers is used to validate not change the values if the user did not set a default. This should stay as true.